### PR TITLE
Improvement/aside suggestions

### PR DESCRIPTION
**Pull request by PeterDMadrid**
**OVERVIEW**

- limits suggested user into 10
- added UI when there no suggested users list

SNIPPETS
No suggested users:
image

Has suggested users (scrollable up to 10)
image

### CODE REVIEW

File: src/app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -62,19 +62,33 @@ public function viewPost(Post $post): View
     /**
      * Like Post
      */
-    public function like(Post $post): RedirectResponse
+    public function like(Post $post)
```
Comment (hipe01mark) [inline src/app/Http/Controllers/PostController.php]: return type!!!

File: src/app/Http/Controllers/PostController.php [hunk 2]
```diff
@@ -62,19 +62,33 @@ public function viewPost(Post $post): View
...
+        ]);
     }
-
+    
     /**
-     * Unlike Post
+     * UnLike Post
      */
-    public function unlike(Post $post): RedirectResponse
+    public function unlike(Post $post)
```
Comment (hipe01mark) [inline src/app/Http/Controllers/PostController.php]: return type!!!

File: src/app/Http/Controllers/PostController.php [hunk 3]
```diff
@@ -62,19 +62,33 @@ public function viewPost(Post $post): View
     /**
      * Like Post
      */
-    public function like(Post $post): RedirectResponse
+    public function like(Post $post)
     {
         $this->postService->like($post);
-        return redirect()->back();
+        $likeUsers = $post->likes->take(10)->map(function ($like) {
```
Comment (mikco-hipe) [inline src/app/Http/Controllers/PostController.php]: convert 10 to constant

File: src/app/Http/Controllers/PostController.php [hunk 4]
```diff
@@ -62,19 +62,33 @@ public function viewPost(Post $post): View
...
+    
     /**
-     * Unlike Post
+     * UnLike Post
      */
     public function unlike(Post $post): RedirectResponse
     {
         $this->postService->unlike($post);
-        return redirect()->back();
+        $likeUsers = $post->likes->take(10)->map(function ($like) {
```
Comment (mikco-hipe) [inline src/app/Http/Controllers/PostController.php]: convert 10 to constant

File: src/public/css/cup.css [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    left: 50%;
+    transform: translate(-50%, -50%) scale(0.25);
+}
+
+.steam {
+    position: absolute;
+    height: 150px;
+    width: 150px;
+    border-radius: 50%;
+    background-color: #292929; /* Darker color for steam */
```
Comment (mikco-hipe) [inline src/public/css/cup.css]: remove comment

File: src/public/css/cup.css [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+#cup {
+    z-index: 1;
+}
+
+#cup-body {
+    position: absolute;
+    height: 200px;
+    width: 300px;
+    border-radius: 0 0 150px 150px;
+    background-color: #555; /* Darker color */
```
Comment (mikco-hipe) [inline src/public/css/cup.css]: remove comment

File: src/public/css/cup.css [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+    display: inline-block;
+    overflow: hidden;
+    z-index: 1;
+}
+
+#cup-shade {
+    position: relative;
+    height: 300px;
+    width: 200px;
+    background-color: #444; /* Darker color */
```
Comment (mikco-hipe) [inline src/public/css/cup.css]: remove comment

File: src/public/css/cup.css [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    transform: rotate(50deg);
+    z-index: 1;
+}
+
+#cup-handle {
+    position: relative;
+    height: 75px;
+    width: 80px;
+    border-radius: 0 150px 150px 0;
+    border: 15px solid #444; /* Darker color */
```
Comment (mikco-hipe) [inline src/public/css/cup.css]: remove comment

File: src/public/css/cup.css [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+    display: inline-block;
+    z-index: 0;
+}
+
+#saucer {
+    position: absolute;
+    height: 30px;
+    width: 300px;  
+    border-radius: 0 0 100px 100px;
+    background-color: #666; /* Darker color */
```
Comment (mikco-hipe) [inline src/public/css/cup.css]: remove comment

File: src/public/js/like-ajax.js [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+                });
+
+                if (likesCount > 10) {
+                    const moreElement = document.createElement('div');
+                    moreElement.textContent = `and ${likesCount - 10} more...`;
+                    likeUsersElement.appendChild(moreElement);
+                }
+            }
+
+            // Toggle the like button class and text
```
Comment (mikco-hipe) [inline src/public/js/like-ajax.js]: remove comment

**Review by mikco-hipe — COMMENTED** [review]
Please check!

File: src/public/css/cup.css [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+
+@-webkit-keyframes steam1 {
+    0% {transform: translateY(0) translateX(0) scale(0.25); opacity: 0.2;}
+    100% {transform: translateY(-200px) translateX(-20px) scale(1); opacity: 0;}
+}
+
+@keyframes steam1 {
+    0% {transform: translateY(0) translateX(0) scale(0.25); opacity: 0.2;}
+    100% {transform: translateY(-200px) translateX(-20px) scale(1); opacity: 0;}
+}
```
Comment (mikco-hipe) [inline src/public/css/cup.css:38]: Is there a way to combine this? they are basically the same

File: src/public/css/cup.css [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+
+@-webkit-keyframes steam2 {
+    0% {transform: translateY(0) translateX(0) scale(0.25); opacity: 0.2;}
+    100% {transform: translateY(-200px) translateX(20px) scale(1); opacity: 0;}
+}
+
+@keyframes steam2 {
+    0% {transform: translateY(0) translateX(0) scale(0.25); opacity: 0.2;}
+    100% {transform: translateY(-200px) translateX(20px) scale(1); opacity: 0;}
+}
```
Comment (mikco-hipe) [inline src/public/css/cup.css:58]: this also

**Review by mikco-hipe — COMMENTED** [review]
Please check!

**Review by mikco-hipe — APPROVED** [review]
LGTB 🙇

**Pull request merged**